src/Validators/AddressValidator.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\FunValidators\Validators;
+
+use WMDE\FunValidators\ConstraintViolation;
+use WMDE\FunValidators\ValidationResult;
+
+class AddressValidator {
```

Comment:
Maybe mention that we are aware of https://www.mjt.me.uk/posts/falsehoods-programmers-believe-about-addresses/ and https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ but only need to validate Addresses from European countries that adhere to our function signatures.